wpilibj/src/main/java/edu/wpi/first/wpilibj/GenericHID.java

```diff
@@ -352,10 +352,10 @@ public String getName() {
   }
 
   /**
-   * Get the axis type of a joystick axis.
+   * Get the axis type of the provided joystick axis.
    *
    * @param axis The axis to read, starting at 0.
-   * @return the axis type of a joystick axis.
+   * @return the axis type of joystick axis.
```

Comment:
This doesn't read well - `the axis type of the given joystick axis` would read better